client/app/states/dashboard/dashboard.state.js

```diff
@@ -35,34 +35,27 @@
   }
 
   /** @ngInject */
-  function resolvePendingRequests(CollectionsApi, $state) {
-    if (!$state.navFeatures.requests.show) {
-      return undefined;
-    }
-
-    var options = {expand: false, filter: ['approval_state=pending'] };
```

Comment:
@h-kataria Just wondering why the RBAC checks have been removed here and in the few places below.